sharded_queue/api.lua

```diff
@@ -217,17 +217,17 @@ function sharded_tube.release(self, task_id, options)
     local bucket_id, _ = utils.unpack_task_id(task_id, bucket_count)
 
     options = options or {}
+    if options.priority == nil and options.pri ~= nil then
+        options.priority = options.pri
+    end
+    options.task_id = task_id
+    options.tube_name = self.tube_name
+
     options.extra = {
         log_request = utils.normalize.log_request(options.log_request) or self.log_request,
     }
 
-    local task, err = vshard.router.call(bucket_id, 'write', 'tube_release', {
-        {
-            tube_name = self.tube_name,
-            task_id = task_id,
-            options = options,
-        }
-    })
+    local task, err = vshard.router.call(bucket_id, 'write', 'tube_release', {options})
```

Comment:
I think `tube_name` and `task_id` should not be packed into `options`, since it contradicts the way they are passed in the other methods.

Reply:
Yeah, I agree with you. But thats how we are passing in other methods. For example if you look at api PUT [method](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/api.lua#L52) 

In the storage before passing it to driver method we are [appending](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/storage.lua#L91) the  tube default options. 

Please let know your thoughts? so we update it.

Reply:
See, for `put` task_id is optional, and `tube_name` is a "hidden" parameter. But for the other methods task_id is not optional, and so for `release`. Leaving alone this inconsistency, there is also a mistake, where `extra` was passed in `options.options`, but now it is in `options`. I suggest leaving the parameters as they are in `delete`, just adding the priority.

Reply:
If we add only priority key still it's going to overwrite the entire `options` key [here](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/storage.lua#L91). 
Currently I am using the custom storage driver, In that I need to support custom option. But I can't pass it using options, even `extra` parameter also got overwritten.  I think the best way is to rename the options key [here](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/storage.lua#L91) as `default_options`.

So that we can pass request options using options key, It will never get overwrite. So in the storage driver we can [check](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/drivers/fifottl.lua#L234) like this.

`local ttl = args.options.ttl or args.default_options.ttl or time.MAX_TIMEOUT`

we can keep the options in separate key without merging it to args and in feature its easy to add more options without changing api method.  please let me know thoughts.

Reply:
You may merge the passed options with `extra` and pass it this way. Either in the same way, as it is done for `log_request`, or by a simpler merge.

Reply:
@akudiyar So you want to pass it as `extra.options.ttl` or `extra.ttl`?  is this only for `release` method?